### PR TITLE
Versioning doc

### LINKED ISSUE #1120: Create a guide on how to use wiki versioning
**Is your feature request related to a problem? Please describe.**

We should mention our version and branching structure in the wiki

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

I created a horribly written guide for the versioning 😆 I added it to your How-Tos although we could also make it a README. If we keep it in the Wiki we should decide where this whole common section should be moved which we need to do anyway.

Should be merged after #1235 

## Links to any relevant issues

Fixes #1120.

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by lucas-tortora** [comment]
I think we can probably keep this in the main README file. If you want I can go through the text and edit it a bit.

File: common/community/contribute-to-wiki/how_tos/versioning.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+keywords:
+  - how to
+description: How to create and maintain your versioned docs.
+---
+
+# Versioning
+
+The Wiki allows you to create your docs. This How-To will explain how it works, how to add a new version, and how to maintain versions.
```
Comment (lucas-tortora) [inline common/community/contribute-to-wiki/how_tos/versioning.md:9]:
```suggestion
This guide will explain how you can add documentation and how to [add](#add-a-version) and [tag](#tag-a-version) versions.
```

File: common/community/contribute-to-wiki/how_tos/versioning.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Add Version
```
Comment (lucas-tortora) [inline common/community/contribute-to-wiki/how_tos/versioning.md:11]:
```suggestion
## Add a Version
```

File: common/community/contribute-to-wiki/how_tos/versioning.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+You can copy an old version or copy from another project and edit everything to your needs.
+
+:::
+
+After that, open the `versionedConfig.js` file, look for your project config, and add a new version in the following format:
+```js
+{
+  label: '2.0',
+  badges: ['IOTA', 'Shimmer'],
```
Comment (lucas-tortora) [inline common/community/contribute-to-wiki/how_tos/versioning.md:25]:
```suggestion
 // the version number 
  label: '2.0.0',
  // Which Networks is your version is compatible with 
  badges: ['IOTA', 'Shimmer'],
```

File: common/community/contribute-to-wiki/how_tos/versioning.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  // You can also add a banner that will show up on all pages above the title.
+  //Example:
+  //bannerPath: path.resolve(
+  //  __dirname,
+  //  'banners/some_nice_markdown.mdx',
+  //),
+},
+```
+
+## Tag Version
```
Comment (lucas-tortora) [inline common/community/contribute-to-wiki/how_tos/versioning.md:36]:
```suggestion
## Tag a Version
```

**Review by lucas-tortora — APPROVED** [review]

File: common/community/contribute-to-wiki/how_tos/versioning.md (hunk 3 above)
Comment (Dr-Electron) [reply common/community/contribute-to-wiki/how_tos/versioning.md:25]: Removed the patch version
```suggestion
  // The version number 
  label: '2.0',
  // Which Networks is your version is compatible with
  badges: ['IOTA', 'Shimmer'],
```

**Review by lucas-tortora — APPROVED** [review]

**Pull request merged**